sre-operational-fabric.md

```diff
@@ -0,0 +1,71 @@
+# (Remote) Senior Software Developer - Rackspace Private Cloud
+
+As a Rackspace Private Cloud (RPC) Operational Fabric SRE, you will help
+support our organizational vision: To become the preferred brand for consuming
+complex open source infrastructure, as-a-Service, delivering industry-leading
+simplicity, efficiency, and reliability.
+
+Our team supports this mission by improving
+[observability](https://en.wikipedia.org/wiki/Observability) (i.e. insight
+into) and [controllability](https://en.wikipedia.org/wiki/Controllability)
+(i.e. state mutation) of cloud deployments located in data-centers throughout
```

Comment:
nit: change data-centers to datacenters or data centers

http://www.dictionary.com/browse/datacenter?s=t